core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -599,6 +601,28 @@ protected void onModified() throws IOException {
     private int slaveAgentPort = SystemProperties.getInteger(Jenkins.class.getName()+".slaveAgentPort",0);
 
     /**
+     * The TCP agent protocols that are explicitly disabled (we store the disabled ones so that newer protocols
+     * are enabled by default).
+     *
+     * @since FIXME
+     */
+    private String disabledAgentProtocols;
```

Comment:
So that the element is not present when not configured... I'll see if I can bash Xstream into a nicer config without using a string